app/src/main/java/model/objects/movingObject/MovingObject.java

```diff
@@ -1,9 +1,36 @@
 package model.objects.movingObject;
 
+
+import lombok.Getter;
+import lombok.Setter;
+import tools.Vector2D;
+
 import java.awt.*;
 
 public class MovingObject extends Creature {
+    protected static final int SPEED_COEFFICIENT = 1000;
```

Comment:
Speed coeffisient ? czyzby to nie jest jakies przspieszenie czy nie wiem

Reply:
Nie rozumiem pytania. To stała, która pozwali mnożyć obecną prędkość i przesuwać obiekty. Możesz dzięki temu szybko zrobić modyfikację jak stwierdzisz, że coś porusza się za szybko, albo za wolno.

Reply:
Za chwilę wyślę dodatek na Vector i będzie widać jak z tego korzystamy.